script/Ciel/Day 12/12_01_CIEL12_5.txt

```diff
@@ -751,22 +797,28 @@ I don't know.
 [sha:215359c9947768eae802552f5ede243afb5f90b7]{
 -- Page 255, Offset 31642.
 -- わからないまま、狂ったように鍵をかけ続ける。
-But until I do, I continue to try to lock the door like I've gone mad.
+Despite not knowing, I keep trying to lock the door like a man possessed.
 }
 [sha:a96b242be1f8db900df31ce443635a4dc7294a2a]{
 -- Page 256, Offset 31643. Mods: @n.
 -- 回っているのはドアノブだけだった。俺の指はノブを掴んだままどうしても離れてくれない。狂ったようにノブを回す。ガチャガチャガチャガチャ。耳障りな音が鼓膜から脳にどくどくと注ぎこまれる。ガチャガチャガチャ、
-The only thing that's turning is the doorknob. My fingers are still gripping the doorknob, refusing to let go. I turn it like I've lost my mind. Clatter, clatter, clatter. The harsh sounds gush into my brain through my eardrums. Clatter, clatter, clatter―――
+The only thing that's turning is the doorknob.
+ My fingers are still gripping the doorknob, refusing to let go.
+ I turn it over and over again like a madman.
+ %{i}Rattle%{/i}, %{i}rattle%{/i}, %{i}rattle%{/i}.
+ That grating sound beats against my eardrums and cascades into my brain.
+ %{i}Rattle%{/i}, %{i}rattle%{/i}, %{i}rattle%{/i}――
```

Comment:
Well, Japanese likes onomatopoeia, but English doesn't. This might be an alternative.
>It turns, and turns, and turns, and turns.
The grating sound of the turning beats against my eardrums, seeping into my brain.
It turns, and turns, and turns, and turns.

Reply:
Keeping the second one for ガチャガチャガチャ、ザク。